driver/normalizer/parser.go

```diff
@@ -25,9 +25,25 @@ var ToNoder = &native.ObjectToNoder{
 	//      children node properties.
 }
 
+// transUASTParserBuilder creates a parser that transform source code files into *uast.Node
```

Comment:
Docstring still has the old function name. I'd just remove it, it's just redundant and error-prone.